### DISCUSSION TITLE
Feature Request: Option for Mandatory VAT ID

### DISCUSSION THREAD

**Discussion by pedroarinto** [pretix/pretix#3940] [Ideas]
We use Pretix for Educational sessions that require VAT ID in Invoices to be accepted as expenses by the state.
Although we did a custom translation of the VAT ID helper text stating that the field is mandatory if the customer wants an invoice for this purpose, most customers do not fill in the field because it is not required to submit the form, although when asked later they say they need their VAT ID in the invoice (I guess no one reads helper texts anymore).
It would be nice if there was an option to make this field mandatory in invoicing details.

**Comment by raphaelm** [comment]
I have never encountered a scenario where making a VAT ID mandatory really makes sense. There are many entities operating inside and outside of Europe that do not have a VAT ID, even though they are commercial or institutional entities. I therefore do not think we'll add a feature for this. However, you can somehow emulate it through a custom tax rule that rejects orders without a VAT ID.
